README.md

```diff
@@ -43,8 +43,9 @@ The recommended way to create a registry is
 using LocalRegistry
 create_registry(name, repository_url; description = "My private registry", push = true)
 ```
-where `name` is the name of your registry and `repository_url` points
-to an *empty* upstream repository where you will host your registry.
+where `name` is the name of your registry as a `String` and
```

Comment:
What alternative is there to using a `String`?

Reply:
Maybe that was just because I was confused because the other example didn’t work, but it wasn’t entirely clear to me what these variables were supposed to be. E. g. `repository_url` could be a `RegistrySpec`, or some special `URL` type.